ssas/systems.go

```diff
@@ -7,16 +7,23 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"time"
+
 	"io"
 	"io/ioutil"
 	"log"
-	"net/url"
 
 	"github.com/jinzhu/gorm"
+	"github.com/pborman/uuid"
 )
 
 const DEFAULT_SCOPE = "bcda-api"
```

Comment:
this and the next const need to be set from environment variables.

Reply:
Do you mean `DEFAULT_SCOPE` and `CREDENTIAL_EXPIRATION`? `DEFAULT_SCOPE` was already there, and `CREDENTIAL_EXPIRATION` is from copying over changes from David's PR. They should be fixed elsewhere and I'll merge them in.